File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.py
def solution(my_string):
    s=my_string.split()
    r=int(s[0])
    for i in range(1,len(s),2):
        if s[i]=='+':
            r+=int(s[i+1])
        else:
            r-=int(s[i+1])
    return r

File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.cpp
#include <string>
#include <vector>
#include <sstream>

using namespace std;

int solution(string my_string) {
    int r=0;
    stringstream ss(my_string);
    string buffer;
    vector<string> op;
    vector<int> num;
    while(getline(ss,buffer,' ')) {
        if(atoi(buffer.c_str()))
            num.push_back(stoi(buffer));
        else
            op.push_back(buffer);
    }
    r=num[0];
    for(int i=0;i<op.size();i++)
        r=op[i]=="+"?r+num[i+1]:r-num[i+1];
    return r;
}